Library/ViewModels/ProjectPamphletViewModel.swift

```diff
@@ -68,7 +70,22 @@ ProjectPamphletViewModelOutputs {
     let refTag = self.refTagProperty.signal
       .map { $0.map(cleanUp(refTag:)) }
 
-    self.configureChildViewControllersWithProject = Signal.combineLatest(project, refTag)
+    let liveStreamEventsFetch = project
+      .take(first: 1)
+      .switchMap { project -> SignalProducer<LiveStreamEventsEnvelope, NoError> in
+        AppEnvironment.current.liveStreamService
+          .fetchEvents(forProjectId: project.id, uid: AppEnvironment.current.currentUser?.id)
+          .ksr_delay(AppEnvironment.current.apiDelayInterval, on: AppEnvironment.current.scheduler)
+          .demoteErrors()
+      }
+
+    let liveStreamEvents = Signal.merge(
+      self.viewDidLoadProperty.signal.flatMap { SignalProducer(value: []) },
+      liveStreamEventsFetch.map { $0.liveStreamEvents }
+    )
+
+    self.configureChildViewControllersWithProjectAndLiveStreams =
+      Signal.combineLatest(project, liveStreamEvents, refTag)
```

Comment:
This is the `combineLatest` that I mentioned in the PR. This emits quite a few times and each will cause a reload of the datasource and table view, perhaps we can buffer/throttle it in some way?